Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1334,19 +1335,23 @@ void MainWindow::showSceneContextMenu(const QPoint& p) {
           CGAL::Three::Scene_item* item = scene->item(index);
           if(!item)
             continue;
-          QVector<QString> action_names;
-          Q_FOREACH(QAction* action, item->contextMenu()->actions())
-          {
-            action_names.push_back(action->text());
```

Comment:
Was this useless?

Reply:
The variable `action_names` was not used.

Reply:
I thought I replied. It was a remain of an earlier attempt for displaying actions for a multiple selection of items that survived the first clean-up.